probe/probe.go

```diff
@@ -533,6 +534,11 @@ func printKubeArmorProbeOutput(buf []byte) error {
 	data = append(data, []string{" ", "Container Security:", green(strconv.FormatBool(kd.ContainerSecurity))})
 	data = append(data, []string{" ", "Container Default Posture:", green(kd.ContainerDefaultPosture.FileAction) + itwhite("(File)"), green(kd.ContainerDefaultPosture.FileAction) + itwhite("(Capabilities)"), green(kd.ContainerDefaultPosture.NetworkAction) + itwhite("(Network)")})
 	data = append(data, []string{" ", "Host Default Posture:", green(kd.HostDefaultPosture.FileAction) + itwhite("(File)"), green(kd.HostDefaultPosture.CapabilitiesAction) + itwhite("(Capabilities)"), green(kd.HostDefaultPosture.NetworkAction) + itwhite("(Network)")})
+	hv := strings.Split(kd.HostVisibility, ",")
+	for i := len(hv) - 1; i < 4; i++ { // Possible values : none/process,file,network,capabilities
+		hv = append(hv, "")
+	}
+	data = append(data, []string{" ", "Host Visibility:", green(hv[0]), green(hv[1]), green(hv[2]), green(hv[3])})
```

Comment:
I think it should be fine if we just mention directly it as a comma seperated  string. We don't need to have tab spaced  strings here. WDYT?

Reply:
I agree, if the format with a comma separated stings looks consistent with the probe output; it would be much simpler.
Does the below output look good ? The more formatted fancy view (tab spaced) would be in the initial text of this draft PR.

![image](https://user-images.githubusercontent.com/1323423/225453854-1683ea41-0280-4409-9c4c-551b3974c494.png)

![image](https://user-images.githubusercontent.com/1323423/225454079-086b9ce7-fa4a-464b-b4b6-da257bed40f4.png)

Reply:
The reason  I suggest this is because we accept comma seperated  values in annotations or flags too  for visibility. This LGTM can you push the changes?

Reply:
Thank you for the feedback, have made the change and pushed the changes.